### DISCUSSION TITLE
How to change flash palette

### DISCUSSION THREAD

**Discussion by horiondreher** [catppuccin/nvim#805] [Q&A] [answered]
Hello!

I'm using catppuccin with LazyVim, and I'm finding it great. But because I use flash a lot, I prefer the strong default colors of it, as it makes it easier to find the keys. 

So, I disabled the lazyvim default integration with it in opts, but it didnt go back to the default colors. How could I fix this?

This is my lazyvim config for catpppucin:

```lua
return {
	{
		"catppuccin/nvim",
		lazy = false,
		priority = 1000,
		name = "catppuccin",
		opts = {
			transparent_background = true,
			flavour = "mocha",
			integrations = {
				flash = false,
			},
		},
	},
	{
		"LazyVim/LazyVim",
		opts = {
			colorscheme = "catppuccin",
		},
	},
}
```

And this is flash with `flash = false`
![Captura de Tela 2024-11-19 às 18 45 13](https://github.com/user-attachments/assets/9f24cd81-12e9-4df2-bfa5-ee824aae663a)

These are the default colors of flash
![247490350-379cb2de-8ec3-4acf-8811-d3590a5854b6](https://github.com/user-attachments/assets/ff13f290-4aae-4355-ba1b-9cb4059e5960)

Disabling transparent_background didn't work.

**Comment by RayGuo-ergou** [comment, marked as answer]
Those are not flash default color, they are from [tokoy night color schema](https://github.com/folke/tokyonight.nvim/blob/45d22cf0e1b93476d3b6d362d720412b3d34465c/extras/lua/tokyonight_moon.lua#L911) which folke uses.

You can customize the highlight groups via option `custom_highlights` or `highlight_overrides`

see: https://github.com/catppuccin/nvim/blob/main/README.md?plain=1#L191-L237
